File: C_module/Assignment 9/Q3.c
#include <stdio.h>

void sum(int*,int);
void main() {
    int n;
    printf("Size of array :: ");
    scanf("%d",&n);
    int arr[n];
    printf("Enter array Elements :: ");
    for(int i=0;i<n;i++){
        scanf("%d",&arr[i]);
    }

    printf("\nElements are :: ");
    for(int i=0;i<n;i++){
        printf("%d ",arr[i]);
    }
    sum(arr,n);
    
}

void sum(int *arr,int n){
    int sum=0;
    for(int i =0;i<n;i++) sum+=arr[i];
    printf("\nSum is :: %d",sum);
}

File: Core_Java_module/Practice/9-16-2025 (Array Practice)/Q3.java
//Find sum of all numbers.

import java.util.Scanner;
class Q3{
	public static void main(String[] args){
		int n,sum=0;
		Scanner sc=new Scanner(System.in);
		System.out.print("Enter array size : ");
		n=sc.nextInt();
		int[] arr=new int[5];
		
		for(int i=0;i<n;i++){
			System.out.print("Enter "+(i+1)+" Element of Array : ");
			arr[i]=sc.nextInt();
		}		
		
		System.out.print("Array Elements are :");
		for(int i=0;i<n;i++){
			System.out.print(arr[i]+" , ");
		}
		System.out.println();

		for(int i=0;i<n;i++){
			sum+=arr[i];
		}
		System.out.println("Sum is : "+sum);
	}
}